src/component/actions/Action/Action.tsx

```diff
@@ -1,21 +1,14 @@
 import * as React from 'react';
-import { FlowDefinition, Node, Group, AnyAction } from '../../flowTypes';
-import { Endpoints } from '../../flowTypes';
-import ComponentMap from '../../services/ComponentMap';
-import { LocalizedObject } from '../../services/Localization';
-import TitleBar from '../TitleBar';
-import {
-    typeConfigListPT,
-    operatorConfigListPT,
-    getTypeConfigPT,
-    getOperatorConfigPT,
-    endpointsPT
-} from '../../providers/ConfigProvider/propTypes';
-import { ConfigProviderContext } from '../../providers/ConfigProvider/configContext';
-import { NodeEditorProps } from '../NodeEditor/NodeEditor';
-import { Language } from '../LanguageSelector';
-
-import * as shared from '../shared.scss';
+import { react as bindCallbacks } from 'auto-bind';
```

Comment:
nesting `Action` within `/actions`. `Action` uses the 'render props' or 'function-as-child' pattern to pass common functionality to other components within the `/actions` directory without returning a _new_ component like a HOC would.